src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -232,59 +270,66 @@ export const BorrowInput = ({
         <Collapse in={true}>
           <LoanSummary
             collateralAssetId={collateralAssetId}
+            collateralAccountId={collateralAccountId}
             depositAmountCryptoPrecision={cryptoDepositAmount ?? '0'}
             borrowAssetId={borrowAsset?.assetId ?? ''}
+            borrowAccountId={borrowAccountId}
           />
         </Collapse>
-        {!isLendingQuoteError && (
-          <CardFooter
-            borderTopWidth={1}
-            borderColor='border.subtle'
-            flexDir='column'
-            gap={4}
-            px={6}
-            py={4}
-            bg='background.surface.raised.accent'
-            borderBottomRadius='xl'
+        <CardFooter
+          borderTopWidth={1}
+          borderColor='border.subtle'
+          flexDir='column'
+          gap={4}
+          px={6}
+          py={4}
+          bg='background.surface.raised.accent'
+          borderBottomRadius='xl'
+        >
+          <Row fontSize='sm' fontWeight='medium'>
+            <Row.Label>{translate('common.slippage')}</Row.Label>
+            <Row.Value>
+              <Skeleton isLoaded={!isLendingQuoteLoading}>
+                <Amount.Crypto
+                  value={lendingQuoteData?.quoteSlippageBorrowedAssetCryptoPrecision ?? '0'}
+                  symbol='BTC'
+                />
+              </Skeleton>
+            </Row.Value>
+          </Row>
+          <Row fontSize='sm' fontWeight='medium'>
+            <Row.Label>{translate('common.gasFee')}</Row.Label>
+            <Row.Value>
+              <Skeleton isLoaded={!(isEstimatedFeesDataLoading || isLendingQuoteLoading)}>
+                <Amount.Fiat value={estimatedFeesData?.txFeeFiat ?? '0'} />
+              </Skeleton>
+            </Row.Value>
+          </Row>
+          <Row fontSize='sm' fontWeight='medium'>
+            <Row.Label>{translate('common.fees')}</Row.Label>
+            <Row.Value>
+              <Skeleton isLoaded={!isLendingQuoteLoading}>
+                <Amount.Fiat value={lendingQuoteData?.quoteTotalFeesFiatUserCurrency ?? '0'} />
+              </Skeleton>
+            </Row.Value>
+          </Row>
+          <Button
+            size='lg'
+            colorScheme={
+              !isLendingQuoteLoading && !isEstimatedFeesDataLoading && isLendingQuoteError
```

Comment:
Or this won't handle not enough funds flow
```suggestion
              !isLendingQuoteLoading && !isEstimatedFeesDataLoading && (quoteErrorTranslation || isLendingQuoteError)
```

Reply:
1551ba67919e34278990d96597b2ca39530dbdd3